include/APIClient/Router.hpp

```diff
@@ -5,24 +5,27 @@
 
 using PutFormData = std::map<std::string, std::string>;
 
+namespace APIClient {
 class Router {
 public:
+    enum class Route : uint8_t {
+        getAllVulkanInfo,
+        getVulkanInfoWithId,
+        putVulkanInfo
+    };
+
+    static constexpr const char* baseAPIUri = LUGBENCH_API_URI;
+    static constexpr const char* apiVersion = LUGBENCH_API_VERSION;
+
     Router() = default;
 
     Router(const Router&) = delete;
-    Router(Router&&) = delete;
 
     Router& operator=(const Router&) = delete;
     Router& operator=(Router&&) = delete;
 
     ~Router() = default;
 
-    enum class Route : uint8_t {
-        getAllVulkanInfo,
-        getVulkanInfoWithId,
-        putVulkanInfo
-    };
-
-public:
-    static constexpr const char* baseAPIUri = LUGBENCH_API_URI;
+    std::string getUrlString(Route route, uint16_t id);
```

Comment:
So, you are using a method that takes an `id` even if just one of the three endpoints take one? I think that something is wrong with that but I'm not sure how to resolve this one.